templates/zerver/integrations.html

```diff
@@ -1243,7 +1243,27 @@
       <p><img class="screenshot" src="/static/images/integrations/librato/009.png" /></p>
 
     </div>
+    <div id="mailchimp" class="integration-instructions">
+      <p>A Zulip fixture for Mail Chimp!</p>
 
+        <p>Before using this integation you will need to create a new stream called <code>mailchimp</code>. To do this go to the panel on the left side with all you streams listed. Then click the gear icon next to <code>STREAMS</code> which says <code>Subscribe, add, or configure streams</code>. Now click the blue button to the left of the search bar which says <code>Create new stream</code> then set the stream name to <code>mailchimp</code> and click <code>Create</code> in the bottom right corner.</p>
+
+        <p>Next, on your {{ settings_html|safe }}, create a Mail Chimp bot.  Construct the URL for
+        the bot using the API key:
+          <code>{{ external_api_uri_subdomain }}/v1/external/mailchimp?api_key=<b>key</b></code>
+        then setup Mail Chimp using these tutorials <a href="http://kb.mailchimp.com/lists/growth/create-a-new-list">Setup list</a> and <a href="http://kb.mailchimp.com/integrations/api-integrations/how-to-set-up-webhooks">Add Webhook</a> (use the url you constructed)
+        </p>
+        <p>To trigger a notication goto the <code>zulip</code> directory through the zulip command line and...</p>
+
+        <p>Run this curl command:</p>
```

Comment:
We probably don't want the uses to manually trigger the notifications. The users shouldn't need access to the zulip repository.

Probably say something like `To trigger a notification to test have someone subscribe or unsubscribe`